GOVERNANCE.md

```diff
@@ -0,0 +1,136 @@
+
+The formal voting process involves the creation of a GitHub issue, pull request, or steering committee mailing list email thread.
+A vote is indicated by specifying `yes` or `no` on the issue, pull request, or mailing list thread.
+After a defined period of time, votes are tallied, and the outcome is published.
+
+Debezium uses several voting mechanisms and procedures for various purposes.
+
+* [Pull request approvals](#pull-request-approvals)
+* [Lazy consensus votes](#lazy-consensus-votes)
+* [Explicit majority votes](#explicit-majority-votes)
+
+Debezium also relies on the notion of **binding** and **non-binding** votes.
+
+A **binding vote** is one cast by specific individuals with given roles based on the voting procedure, who's vote is used to calculate the outcome.
+A **non-binding vote** is one cast by individuals who do not have binding vote roles, and these votes do not count toward the outcome, but are used to guide those with binding votes to a resolution that best fits all involved.
+
+The following describes for each procedure, those who may vote, how the votes are recorded, and duration of the vote.
+
+### Pull request approvals
+
+Pull requests against most repositories may be merged after receiving at least two positive committer votes.
+Voting for pull requests can be done using a comment or approving the pull request on GitHub.
+If the pull request author is a member of the Debezium steering committee, this counts as one of the two required votes.
+
+Pull requests that are small, low-risk, uncontroversial, or prefixed with `[ci]` or `[docs]` may freely be merged by any committer without any votes.
+Whenever in doubt about the pull request's scope and risk, use the two positive vote requirement.
+
+### Lazy consensus votes
+
+Lazy consensus votes use the **+1**, **0**, or **-1** votes where:
+
+* `+1` indicates **yes** or the **affirmative**
+* `-1` indicates **no** or the **negative**
+* `0` indicates you have no specific opinion
+
+The voting process concludes when there are at least three `+1` binding votes, and no `-1` binding votes.
+The vote should be open for at least **TBD** days to allow everyone to participate.
+
+### Explicit majority votes
+
+An explicit majority vote is simply a `+1` or `-1` based vote.
+The vote succeeds when at least **three binding** votes are cast, and two-thirds of cast binding values are `+1`.
+The vote should be open for at least **TBD** days to allow everyone to participate.
+
+## Voting for specific subject areas
+
+The following describes the voting procedures for specific subject areas.
+
+### Governance changes
+
+Except for the steering committee members marking themselves as Emeritus, all changes in the [governance](https://github.com/debezium/governance) repository require an explicit majority vote of steering committee members.
+These votes are held on the steering committee's mailing lists, and the outcome will be publicly published.
+
+### Add/remove steering committee members
```

Comment:
Is the tenure going to be unlimited? What about having defined length of term where the member then needs to be confirmed or another one voted?